messaging/amqp-reactive/src/main/resources/application.properties

```diff
@@ -1,9 +1,6 @@
 quarkus.openshift.expose=true
 quarkus.s2i.base-jvm-image=registry.access.redhat.com/openjdk/openjdk-11-rhel7
 
-# Configures the AMQP broker credentials.
-amqp-username=quarkus
-amqp-password=quarkus
```

Comment:
The broker is configured with security disabled, so this configuration is not needed. Moreover, with this configuration the Openshift test was randomly failing.